src/pudl/transform/eiaaeo.py

```diff
@@ -55,6 +55,67 @@ def get_category_info(category_name: pd.Series) -> pd.Series:
     return fields
 
 
+def subtotals_match_reported_totals_ratio(
```

Comment:
I split the old `check_totals_add_up` function into the two separate operations so that we could use them in different places, and then moved the original function into the body of the asset that it was specifically designed for.